client/cmd/apply/apply.go

```diff
@@ -179,6 +186,7 @@ func (c *applyCommand) printSuccess(namespaceName, operation, kind, name string)
 
 func (c *applyCommand) printFailed(namespaceName, operation, kind, name, cause string) {
 	c.logger.Error("[%s] %s: %s %s ❌", namespaceName, operation, kind, name)
+	c.errors.Append(fmt.Errorf("[%s] %s: %s %s ❌", namespaceName, operation, kind, name))
```

Comment:
since the function name is `printFailed`, lets keep this function only for printing errors?
appending errors can be put whenever a function call returns error